config/service_providers.yml

```diff
@@ -153,17 +153,23 @@ production:
     sp_initiated_login_url: 'https://dashboard.demo.login.gov/users/auth/saml'
     block_encryption: 'aes256-cbc'
     cert: 'identity_dashboard_cert'
+    attribute_bundle:
+      - email
 
```

Comment:
should this be here or on the dashboard side? The SP can explicitly request attributes.